packages/platform-ios/src/commands/runIOS/index.ts

```diff
@@ -441,7 +441,7 @@ function getBuildPath(
 }
 
 function getProductName(buildOutput: string) {
-  const productNameMatch = /export FULL_PRODUCT_NAME="?(.+).app"?$/m.exec(
+  const productNameMatch = /export FULL_PRODUCT_NAME\\="?(.+).app"?$/m.exec(
```

Comment:
Can you make this backslash optional? The regex worked earlier, so that's probably some recent change.